isis/src/base/objs/Sensor/Sensor.cpp

```diff
@@ -574,7 +575,9 @@ namespace Isis {
    * @return @b double The angle of right ascension, in degrees.
    */
   double Sensor::RightAscension() {
-    if (m_newLookB) computeRaDec();
+    if (m_newLookB) {
+      computeRaDec();
+    }
     return m_ra;
   }
```

Comment:
computeRaDec should be called if the look vector has changed, indicated by m_newLookB being true. It's basically a work around to avoid computing the RA and Dec everytime the camera model is set to a new thing.